static/app/views/performance/transactionSummary/transactionOverview/sidebarCharts.tsx

```diff
@@ -233,7 +234,7 @@ function SidebarCharts({
             {({results, errored, loading, reloading}) => {
               if (errored) {
                 return (
-                  <ErrorPanel height="580px">
+                  <ErrorPanel height="480px">
```

Comment:
Any particular reason why these changed to 480px other than "we want them smaller"?

Reply:
The rendered graphs are actually 480px. Unsure why it's 580px here as that causes additional layout shifts as the graph loads. You can check out the visual snapshots to see the what I mean.

Reply:
Ah, I checked it out locally and saw the difference, thanks for pointing that out!